### PR TITLE
[ETFE-3981] - Read in notification of diverted movement (IE803)

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: test/uk/gov/hmrc/emcstfe/fixtures/GetMovementFixture.scala [hunk 1]
```diff
@@ -369,26 +370,31 @@ trait GetMovementFixture extends BaseFixtures with TraderModelFixtures {
...
+                                               |              <urn:NotificationType>1</urn:NotificationType>
+                                               |              <urn:NotificationDateAndTime>2024-06-05T00:00:01</urn:NotificationDateAndTime>
+                                               |              <urn:AdministrativeReferenceCode>20GB00000000000341760</urn:AdministrativeReferenceCode>
+                                               |              <urn:SequenceNumber>1</urn:SequenceNumber>
+                                               |            </urn:ExciseNotification>
+                                               |            <urn:DownstreamArc>
+                                               |              <urn:AdministrativeReferenceCode>$testArc</urn:AdministrativeReferenceCode>
+                                               |            </urn:DownstreamArc>
+                                               |            <urn:DownstreamArc>
+                                               |              <urn:AdministrativeReferenceCode>${testArc}1</urn:AdministrativeReferenceCode>
```
Comment (grahampaulcook) [inline test/uk/gov/hmrc/emcstfe/fixtures/GetMovementFixture.scala:393]: Appending a 1 to that `testArc` would push it to 22 characters which is 1 over the schema defined limit although this is probably okay for display purposes.

```  <!--==========================================================-->
  <!--===                    Upstream ARC                    ===-->
  <!--==========================================================-->
  <xs:simpleType name="UpstreamArcType">
    <xs:annotation>
      <xs:documentation>
        <doc:description value="Upstream ARC" />
      </xs:documentation>
    </xs:annotation>
    <xs:restriction base="xs:token">
      <xs:length value="21" />
      <xs:pattern value=".{21}" />
    </xs:restriction>
  </xs:simpleType>```

Comment (grahampaulcook) [reply test/uk/gov/hmrc/emcstfe/fixtures/GetMovementFixture.scala:393]: Also I wouldn't have thought a notification type 1 would have downstream arcs populated, isn't that more likely for a notification type 2.

Comment (grahampaulcook) [reply test/uk/gov/hmrc/emcstfe/fixtures/GetMovementFixture.scala:393]: Although we can only have 1 `ie803:IE803` so I suppose it caters for both types of IE803's
image

**Review by grahampaulcook — APPROVED** [review]

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**